src/actions/plugins/easymotion/types.ts

```diff
@@ -1,4 +1,5 @@
 export type LabelPosition = 'after' | 'before';
+export type jumpToAnywhere = true | false;
```

Comment:
Capital here please as we're exposing the type :)

Reply:
OK, updated 👍